src/CardList/CardList.js

```diff
@@ -4,6 +4,10 @@ import './CardList.scss';
 import { useState } from 'react';
 import AddCard from '../AddCard/AddCard';
 import Modal from '../Modal/Modal';
+import WithLoadingDelay from '../common/WithLoadingDelay/WithLoadingDelay';
+
+
+const CardWithLoading = WithLoadingDelay(Card, {height: '200px'});
```

Comment:
Can you please move WithLoadingDelay(...) to export statement of Card component?
And in this file go back to <Card .../>.
Correct it and merge please.